packages/integration-react/src/ScmIntegrationsApi.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Config } from '@backstage/config';
+import { ApiRef, createApiRef } from '@backstage/core';
+import { ScmIntegrations } from '@backstage/integration';
+
+export class ScmIntegrationsApi extends ScmIntegrations {
```

Comment:
This is a bit wonky, I know. But it lets you import only this package into the app itself, and it lets us iterate on the type in the future.

Reply:
Sure, but why? This gives a bit nicer IDE assistance when you refactor and create the class, and conveys intent

Reply:
oh wait, now i backed up a bit - i see - yeah and change the ref to ScmIntegrationRegistry interface